changelog/README.md

```diff
@@ -27,6 +27,11 @@
 * Extended [Voucher code](../operations/vouchercodes.md#voucher-code) response object with `IsActive` property.
   * [Get all voucher codes](../operations/vouchercodes.md#get-all-voucher-codes)
 
+## 30th July 2024

```

Comment:
If I'm interpreting this correctly, we should also add the fact that some additional fields are added to `Update availability blocks` such as `Budget`. I presume these fields were already supported but not in the docs? Either way, they should be mentioned as we should be transparent and open.